lib/Subscribers/ConfirmationEmailMailer.php

```diff
@@ -104,7 +104,7 @@ public function sendConfirmationEmail(Subscriber $subscriber) {
     );
 
     //create a text version. @ is important here, Html2Text throws warnings
-    $text = @Html2Text::convert((mb_detect_encoding($body, 'UTF-8', true)) ? $body : utf8_encode($body));
+    $text = @Html2Text::convert((mb_detect_encoding($body, 'UTF-8', true)) ? $body : utf8_encode($body), true);
```

Comment:
There are two other places where we use `Html2Text::convert` should we update those as well? 
Also, would it be better to format this line a bit differently? 
```
$text = @Html2Text::convert(
  (mb_detect_encoding($body, 'UTF-8', true)) ? $body : utf8_encode($body)
  , true
);
```
Not a big deal, but I think it improves the readability.

Reply:
I updated the code for better readability. 

I don't know what's better in the other two places - being consistent or changing the least of code not to introduce bugs. I decided for the latter, but I'm fine changing it everywhere if you think consistency is more important.